packages/orbit-components/src/ListChoice/index.js.flow

```diff
@@ -10,6 +10,7 @@ export type Props = {|
   +description?: Translation,
   +selectable?: boolean,
   +selected?: boolean,
+  +disabled?: boolean,
   +icon?: React$Node,
```

Comment:
i know, that you add only `disabled` prop, but here is `React$Node` below, you could replace it to `React.Node` too :)

Reply:
✅